src/components/LeftSidebar/NewGroupConversation/SetConversationType/SetConversationType.vue

```diff
@@ -0,0 +1,77 @@
+<!--
+  - @copyright Copyright (c) 2019 Marco Ambrosini <marcoambrosini@pm.me>
+  -
+  - @author Marco Ambrosini <marcoambrosini@pm.me>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+	<div class="conversation-type">
+		<input
+			type="checkbox"
+			:checked="value"
+			@change="handleInput" />
+		<label class="conversation-type__label">{{ t('spreed', 'Make ') }}<strong>{{ conversationDisplayName }}</strong>{{ t('spreed', ' public') }}</label>
```

Comment:
This breaks translations. Assign the following to a computed and use that:

```
 t('spreed', 'Make {name} public', { name: this.conversationDisplayName)
```

Reply:
Also use a full string for the empty name and do the switch in the computed:

```js
if (this.conversationName === '') {
    return  t('spreed', 'Make your conversation public')
}
return  t('spreed', 'Make {name} public', { name: this.conversationName })

Reply:
Can I have this bold as it is right now @nickvergessen ?

Reply:
```js
if (this.conversationName === '') {
    return  t('spreed', 'Make your conversation public')
}
return  t('spreed', 'Make {name} public', { name: '<strong>' + escapeHTML(this.conversationName) + '</strong>' }, undefined, { escape: false })
```

Reply:
Oh that is cool!

Reply:
You will need to use v-html though